tasks/main.yml

```
@@ -1,5 +1,11 @@
 ---
 
+- name: ensure firewalld is installed
+  package:
+    state: latest
```

Please change as suggested by the linter.
```suggestion
    state: present
```